docker/build-and-push-images.yml

```diff
@@ -386,6 +386,7 @@ stages:
             targetType: "inline"
             script: |
               Invoke-RestMethod -Uri "$(DEV_PORTAL)/api/UpdateDefinition?demoType=edge-xm0&sitecoreVersion=$(SITECORE_VERSION)&channel=stable" -Method POST -Body "{ apiKey: '$(DEV_PORTAL_API_KEY)', tag: '$(GitVersion.SemVer)' }" -ContentType "application/json"
+              Invoke-RestMethod -Uri "$(NEW_DEV_PORTAL_UPDATE_DEFINITION)&definitionId=2" -Method POST -Body "{ tag: '$(GitVersion.SemVer)', revision: '$(GitVersion.Sha)' }" -ContentType "application/json"
```

Comment:
Perhaps petty - but can you change "NEW_DEV_PORTAL_UPDATE_DEFINITION" to "DEV_PORTAL_UPDATE_DEFINITION_V2" instead? Same w/ Prod portal variable.

Reply:
DEV_PORTAL_V2_UPDATE_DEFINITION maybe? As this is a new portal, not new endpoint.

Reply:
Works for me

Reply:
Done